notebooks/MAST/astroqueryMAST/beginner_program_id_search/beginner_program_id_search.ipynb

```diff
@@ -0,0 +1,402 @@
+    "\n",
+    "For level 3 observations, it's likely that there are many associated (levels 2 and 1) data products. Let's take a look at how many products are associated with second observation from our search above."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "second_obs = obs_list[1:2]\n",
+    "data_products = Observations.get_product_list(second_obs)\n",
+    "print(len(data_products))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "This produces over 3000 data products associated with this observation! This is not uncommon for a JWST level-3 observation. In the next section, we'll take a look at how we can filter down the number of results before we download them."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Filtering and Downloading Data\n",
+    "\n",
+    "### Filtering"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "You can apply filter keyword arguments to download only data products that meet your given criteria. Available filters are “mrp_only” ([minimum recommended products](https://outerspace.stsci.edu/display/MASTDOCS/Minimum+Recommended+Products)), “extension” (file extension), calib_level (calibration level), and all products fields listed [here](https://mast.stsci.edu/api/v0/_productsfields.html).\n",
+    "\n",
+    "In this example, let's try filtering for only the level 2, calibrated exposures. It is important that we also filter by \"SCIENCE\" type products; otherwise, our results will include guide star acquisition images."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "filtered_prod = Observations.filter_products(data_products, calib_level=[2], productType=\"SCIENCE\")\n",
+    "\n",
+    "# Again, we choose columns of interest for convenience\n",
+    "disp_col = ['obsID','dataproduct_type','productFilename','size','calib_level']\n",
+    "filtered_prod[disp_col].show_in_notebook(display_length=10)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Well, that was effective! We now have 80 files, instead of over 3000.\n",
+    "\n",
+    "As a final check before we proceed to the download, let's find the total file size of our results:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "total = 0\n",
```

Comment:
Here's a cleaner way to get the total in just one line, plus your print statement (no need for a 'for' loop here):

`total_test = sum(filtered_prod['size'])
print('{:.2f}'.format(total_test/10**9), "GB")`